packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -78,7013 +73,9172 @@
-     * \param coordinate_destinations is the output array that holds which part each coordinate should be sent.
-     */
-    void mj_assign_parts_to_procs(
-        mj_gno_t * num_points_in_all_processor_parts,
-        mj_part_t num_parts,
-        mj_part_t num_procs,
-        mj_lno_t *send_count_to_each_proc, //output: sized nprocs, show the number of send point counts to each proc.
-        std::vector<mj_part_t> *next_future_num_parts_in_parts,//input how many more partitions the part will be partitioned into.
-        mj_part_t &out_num_part, //output, how many parts the processor will have. this is always 1 for this function.
-        std::vector<mj_part_t> &out_part_indices, //output: the part indices which the processor is assigned to.
-        mj_part_t &output_part_numbering_begin_index, //output: how much the part number should be shifted when setting the solution
-        int *coordinate_destinations);
-
-    /*! \brief Function fills up coordinate_destinations is the output array
-     * that holds which part each coordinate should be sent. In addition it calculates
-     * the shift amount (output_part_numbering_begin_index) to be done when
-     * final numberings of the parts are performed.
-     *
-     *
-     * \param num_procs is the number of processor attending to migration operation.
-     * \param num_new_local_points is the output to represent the new number of local points.
-     * \param iteration is the string for the current iteration.
-     * \param coordinate_destinations is the output array that holds which part each coordinate should be sent.
-     * \param num_parts is the number of parts that exist in the current partitioning.
-     */
-    void mj_migrate_coords(
-        mj_part_t num_procs,
-        mj_lno_t &num_new_local_points,
-        std::string iteration,
-        int *coordinate_destinations,
-        mj_part_t num_parts);
-
-    /*! \brief Function creates the new subcomminicator for the processors
-     * given in processor_ranks_for_subcomm.
-     *
-     * \param processor_ranks_for_subcomm is the vector that has the ranks of
-     * the processors that will be in the same group.
-     */
-    void create_sub_communicator(std::vector<mj_part_t> &processor_ranks_for_subcomm);
-
-
-    /*! \brief Function writes the new permutation arrays after the migration.
-     *
-     * \param output_num_parts is the number of parts that is assigned to the processor.
-     * \param num_parts is the number of parts right before migration.
-     */
-    void fill_permutation_array(
-        mj_part_t output_num_parts,
-        mj_part_t num_parts);
-
-    /*! \brief Function checks if should do migration or not.
-     * \param current_num_parts is the number of parts in the process.
-     * \param output_part_begin_index is the number that will be used as beginning part number
-     * \param output_part_boxes is the array that holds the part boxes
-     * \param is_data_ever_migrated is the boolean
```

Comment:
We discussed and since the original code had mixed formats, we decided we won't fix all now and just document. I still have general documentation to fix everywhere.